docs/sources/reference/components/discovery.azure.md

```diff
@@ -32,6 +32,7 @@ Name                     | Type                | Description
 `proxy_from_environment` | `bool`              | Use the proxy URL indicated by environment variables.                                            | `false`              | no
 `proxy_connect_header`   | `map(list(secret))` | Specifies headers to send to proxies during CONNECT requests.                                    |                      | no
 `follow_redirects`       | `bool`              | Whether redirects returned by the server should be followed.                                     | `true`               | no
+`host`                   | `string`            | Override HTTP hostname                                                                           |         | no
```

Comment:
```suggestion
`host`                   | `string`            | Override HTTP hostname.                                                                           |         | no
```

For consistency, we should add a full stop (period) at the end of this description.  This should be cascaded to to all instances of `Override HTTP hostname.`